src/pages/community-champions.js

```diff
@@ -42,6 +49,79 @@ const StyledHeroBanner = styled(HeroBanner)`
     }
 `;
 
+const ContentContainer = styled('div')`
+    margin: 0 ${size.xxlarge};
+    @media ${screenSize.upToLarge} {
+        margin: 0 ${size.default};
+    }
+`;
+
+const Title = styled(H1)`
+    margin: ${size.xlarge} 0;
+    text-align: center;
+    @media ${screenSize.upToLarge} {
+        margin-top: ${size.large};
+        margin-bottom: ${size.mediumLarge};
+    }
+`;
+
+const Description = styled(P)`
+    color: ${({ theme }) => theme.colorMap.greyLightTwo};
+    margin-bottom: 0;
```

Comment:
We have a `collapse` prop on `P` that can do this for us without additional CSS, let's add that!
```suggestion
```